src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenOnBehalfOfParameters.cs

```diff
@@ -9,13 +10,21 @@ namespace Microsoft.Identity.Client.ApiConfig.Parameters
     internal class AcquireTokenOnBehalfOfParameters : AbstractAcquireTokenConfidentialClientParameters, IAcquireTokenParameters
     {
         /// <remarks>
-        /// User assertion is null when <see cref="ILongRunningWebApi.AcquireTokenInLongRunningProcess"/> is called.
+        /// User assertion is null when <see cref="ILongRunningWebApi.InitiateLongRunningProcessInWebApi(IEnumerable{string}, string, ref string)"/> is called.
         /// </remarks>
         public UserAssertion UserAssertion { get; set; }
         /// <summary>
         /// User-provided cache key for long-running OBO flow.
         /// </summary>
         public string LongRunningOboCacheKey { get; set; }
+
+        /// <summary>
+        /// Only affects <see cref="ILongRunningWebApi.InitiateLongRunningProcessInWebApi(IEnumerable{string}, string, ref string)"/>.
+        /// When enabled, mimics MSAL 4.50.0 and below behavior - does not check cached tokens based on OBO assertions.
+        /// When disabled (default behavior), cached tokens will only be returned if the OBO assertion in the request matched the assertion of the cached token.
```

Comment:
So the cache will always be searched? The difference is whether the assertion is matched or not. I think if this is the case we can name it something like `SearchInCacheUsingAssertion`?

Reply:
This is an outdated comment. Updated.